source/scss/_config.scss

```diff
@@ -21,8 +21,20 @@
 //  xxl: $bp--xxl
 //) !default;
 
+//bootstrap setup
+@import "~bootstrap/scss/functions";
+@import "~bootstrap/scss/variables";
+@import "~bootstrap/scss/mixins";
+@import "~bootstrap/scss/print";
```

Comment:
Hard rule: `_config.scss` produces **no printing css**.  So we'll cut it above `print`. This means that reboot, type, etc, have to be pulled in elsewhere in our `00-base` folder, or in `design-system.js`.